### DISCUSSION TITLE
I'm wondering how to exclude some frameworks of PySide6 to reduce the executable size

### DISCUSSION THREAD

**Discussion by kaka1909** [pyinstaller/pyinstaller#8673] [Help]
I am developing a PySide6 application and use pyinstaller to package, I want to reduce the executable size, so I was looking for something to delete. And I found in my dist/_internal/PySide6/lib/ , there are many frameworks that i do not need actually, just like "/myproject/dist/dakar/_internal/PySide6/Qt/lib/QtPdf.framework", but I don't know how to exclude it. 

I tried to use exclude in .spec, but it does not work. Need a help. Thanks

-- my spec file.
```
# -*- mode: python ; coding: utf-8 -*-


a = Analysis(
    ['dakar.py'],
    pathex=[],
    binaries=[],
    datas=[
        ('config', 'config'),
    ],
    hiddenimports=['pkg_resources.extern'],
    hookspath=[],
    hooksconfig={},
    runtime_hooks=[],
    excludes=['ipython', 'jedi'],
    noarchive=False,
    optimize=0,
)

pyz = PYZ(a.pure)

exe = EXE(
    pyz,
    a.scripts,
    [],
    exclude_binaries=True,
    name='dakar',
    debug=False,
    bootloader_ignore_signals=False,
    strip=False,
    upx=True,
    console=False,
    disable_windowed_traceback=False,
    argv_emulation=False,
    target_arch=None,
    codesign_identity=None,
    entitlements_file=None,
)
coll = COLLECT(
    exe,
    a.binaries,
    a.datas,
    strip=False,
    upx=True,
    upx_exclude=[],
    name='dakar',
)
app = BUNDLE(
    coll,
    name='my.app',
    icon='my.icns',
    version='1.0.0',
    bundle_identifier=None,
)

**Comment by rokm** [comment]
You would have to filter the entries of `a.datas` and `a.binaries` before they are passed on to `EXE` and `COLLECT`. 

But if you are building an .app bundle, you might as well remove them manually in post-processing, and re-sign the bundle (that would probably be easier to experiment with, as well).
